src/store/Store.ts

```diff
@@ -59,18 +59,23 @@ export interface GraphState {
   } | null;
 }
 
-export interface Token {
+export enum LoginStatus {
+  logging,
+  loggedIn,
+  loggedOut,
+  error
+}
+
+export interface Session {
```

Comment:
Should we be more specific than `Session` there are many different sessions; or is this the `KialiSession`?